src/network.cpp

```diff
@@ -196,103 +196,99 @@ void IRAM_ATTR RemoteLoopEntry(void *)
     while (true)
     {
         g_RemoteControl.handle();
-        delay(20);        
+        delay(20);
     }
 }
 #endif
 
 // ConnectToWiFi
 //
-// Connect to the pre-configured WiFi network.  
+// Connect to the pre-configured WiFi network.
 
 #if ENABLE_WIFI
 
-    #define WIFI_RETRIES 5
-
     bool ConnectToWiFi(uint cRetries)
     {
         static bool bPreviousConnection = false;
 
         // Already connected, Go no further.
-        if (false == WiFi.isConnected())
-        {
-            debugI("Setting host name to %s...%s", cszHostname,WLtoString(WiFi.status()));
+        if (WiFi.isConnected())
+            return true;
 
-            if (WiFi_ssid == "Unset" || WiFi_ssid.length() == 0)
-            {
-                debugW("WiFi Credentials not set, cannot connect");
-                return false;
-            }
+        debugI("Setting host name to %s...%s", cszHostname,WLtoString(WiFi.status()));
 
-            for (uint iPass = 0; iPass < cRetries; iPass++)
-            {
-                Serial.printf("Pass %u of %u: Connecting to Wifi SSID: %s - ESP32 Free Memory: %u, PSRAM:%u, PSRAM Free: %u\n",
-                    iPass, cRetries, WiFi_ssid, ESP.getFreeHeap(), ESP.getPsramSize(), ESP.getFreePsram());
+        if (WiFi_ssid == "Unset" || WiFi_ssid.length() == 0)
+        {
+            debugW("WiFi Credentials not set, cannot connect");
+            return false;
+        }
 
-                WiFi.disconnect();
-                WiFi.mode(WIFI_STA);
-                WiFi.begin(WiFi_ssid.c_str(), WiFi_password.c_str());
+        for (uint iPass = 0; iPass < cRetries; iPass++)
+        {
+            Serial.printf("Pass %u of %u: Connecting to Wifi SSID: %s - ESP32 Free Memory: %u, PSRAM:%u, PSRAM Free: %u\n",
+                iPass + 1, cRetries, WiFi_ssid, ESP.getFreeHeap(), ESP.getPsramSize(), ESP.getFreePsram());

```

Comment:
Come on, no pass 0?  We're all programmers here :-)

Reply:
You're probably right, but "0 to 4 out of 5" still causes a short circuit in my old-fashioned, simple little brain. :)